skopt/searchcv.py

```diff
@@ -398,6 +399,146 @@ def add_spaces(self, names, search_spaces):
         for space, name in zip(search_spaces, names):
             self.search_spaces_[name] = space
 
+    # copied for compatibility with 0.19 sklearn from 0.18 BaseSearchCV
+    @property
+    def best_score_(self):
+        sk_model_sel.check_is_fitted(self, 'cv_results_')
```

Comment:
we should import this as `from sklearn.utils.validation import check_is_fitted` instead of pulling it out of the model selection module.